pkg/operator/ceph/object/admin.go

```diff
@@ -110,8 +111,50 @@ func runAdminCommand(c *Context, expectJSON bool, args ...string) (string, error
 			fmt.Sprintf("--rgw-zone=%s", c.Zone),
 		}
 
-		return RunAdminCommandNoMultisite(c, expectJSON, append(args, options...)...)
+		args = append(args, options...)
 	}
 
-	return RunAdminCommandNoMultisite(c, expectJSON, args...)
+	// work around https://github.com/rook/rook/issues/7573 (FIFO file I/O issue)
+	result, err := RunAdminCommandNoMultisite(c, expectJSON, args...)
+	if err != nil && isFifoFileIOError(err) {
+		command := ""
+		if len(args) >= 2 {
+			command = fmt.Sprintf(" %s %s", args[0], args[1]) // e.g., "user create" or "user update"
```

Comment:
This command is only used for logging? Why bother to log the extra args? Seems like debug level info.

Reply:
I think I can get rid of it...